### PR TITLE
Clean up some comments and unused rabbitmq

### PR DESCRIPTION
**Pull request by t-persson**

### CODE REVIEW

**Comment by t-persson** [comment]
Current dependencies on/for this PR:
* main
  * **PR #54** <a href="https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/54" target="_blank">Graphite</a> 
    * **PR #55** <a href="https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/55" target="_blank">Graphite</a>  (this PR)
      * **PR #56** <a href="https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/56" target="_blank">Graphite</a> 
        * **PR #57** <a href="https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/57" target="_blank">Graphite</a> 
          * **PR #58** <a href="https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/58" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/55?utm_source=stack-comment).

**Review by fredjn — DISMISSED** [review]

File: src/environment_provider/environment_provider.py [hunk 1]
```diff
@@ -122,8 +120,6 @@ def configure(self, suite_id):
             int(os.getenv("ETOS_WAIT_FOR_LOG_AREA_TIMEOUT", "10")),
         )
 
-        self.etos.config.rabbitmq_publisher_from_environment()
```
Comment (t-persson) [inline src/environment_provider/environment_provider.py:125]: This is important and should be re-added!

**Review by andjoe-axis — APPROVED** [review]

**Review by fredjn — DISMISSED** [review]

**Comment by t-persson** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/eiffel-community/etos-environment-provider/55?utm_source=stack-comment).


Job Reference: BlPbstb4LcZJpY2xQRx9

**Review by andjoe-axis — APPROVED** [review]

**Review by fredjn — APPROVED** [review]

**Review by nordixinfra — COMMENTED** [review]
- Build Status: **SUCCESS**
- Build URL: [Nordix Jenkins](https://jenkins.nordix.org/job/etos-environment-provider-build-merge-ubuntu2004-main/18/)
- Command to pull image:
    `docker pull registry.nordix.org/eiffel/etos-environment-provider:dev`
    or
    `podman pull registry.nordix.org/eiffel/etos-environment-provider:dev`

**Pull request merged**